src/main/java/com/onarandombox/MultiverseCore/commands/TeleportCommand.java

```diff
@@ -35,6 +35,10 @@ public void onTeleportCommand(BukkitCommandIssuer issuer,
     ) {
         // TODO Add warning if teleporting too many players at once.
         for (Player player : players) {
+            if (!this.plugin.getPlayerActionChecker().canUseDestinationToTeleport(issuer.getIssuer(), player, destination).isSuccessful()) {
+                issuer.sendMessage("You don't have permission to use this destination.");
+                continue;
+            }
```

Comment:
Should this not be checking `canGoToDestination` like the check command?

Reply:
I'm moving it to ACF command condition handling, but mainly bcu not the main scope of this PR, so I put it here for testing, just to make sure the method logic works.

Reply:
Also, canGoToDestination is already checked in the player teleport event.